packages/microcode/microcode.spec

```diff
@@ -26,11 +26,22 @@ Source3: https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/arc
 
 # Lets us install "microcode" to pull in the AMD and Intel updates.
 Requires: %{_cross_os}microcode-amd
-Requires: %{_cross_os}microcode-intel
+Requires: %{_cross_os}microcode-intel-other
+Requires: %{_cross_os}microcode-intel-ec2
 
 %description
 %{summary}.
 
+%package ec2
+Summary: Microcode for EC2 processors
+URL: https://github.com/bottlerocket-os/bottlerocket/tree/develop/packages/microcode
+Requires: %{_cross_os}microcode-amd
```

Comment:
AMD microcode should get the same treatment, while we're making this change. Either:
1. separate ec2 and other subpackages
2. or a single subpackages that provides the ec2 and other capabilities

Then this would become:
```
Requires: %{_cross_os}microcode-amd-ec2
```

And then the dependencies will be more obviously correct to the reader, rather than making them wonder why AMD is different.

Reply:
Let me check the AMD microcode list against the latest AL srpm again. My recollection is that there are no excess AMD microcode files (that is, the non-EC2 list would be empty).

Reply:
I verified that the "microcode-amd-other" category would be empty. There are five (small) files on the AMD side.

Reply:
I'd still like to see the change I asked for above. Option (2) would avoid an empty package while still letting this change to:
```
Requires: %{_cross_os}microcode-amd-ec2
```

Reply:
Done